src/js/panels.js

```diff
@@ -107,7 +117,6 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
         var step = reverse ? (that.options.step * -1) : that.options.step;
         var newValue = that.model.value + step;
-        newValue = gpii.firstDiscovery.panel.ranged.clip(newValue, that.options.range.min, that.options.range.max);
```

Comment:
as discussed, removed the clip method entirely and simplified the updateMeter function instead.